docs/source/tutorials/extending_mantid_with_python/python_fit_functions/04_exercise_5.rst

```diff
@@ -0,0 +1,36 @@
+.. _04_exercise_5:
+
+==========
+Exercise 5
+==========
+
+The aim of this exercise is to implement a simple linear fitting function
+defined as:
+
+.. math::
+
+    y = A_0 + A_1x
+
+The function should have 2 parameters:
+
+#. ``A0``: intercept with the y-axis.
+#. ``A1``: gradient of the function.
+
+Define a class called ``PyLinearFunction``
+
+* Add an init function that declares the two parameters.
+* Add a function1D function that returns the y values for the given x values
+  as a numpy array.
+* Add the appropriate line to register it with Mantid.
+
+To test:
+
+#. Execute your code in the script window to register the function with Mantid.
+#. Load file *GEM38370_Focussed_Legacy.nxs*.
+#. Plot spectrum number 5.
+#. Activate the fitting tool.
+#. Right click on the plot and select "Add Other Function...".
+#. Select PyLinearFunction.
+#. Click Fit!
```

Comment:
Before Fitting, can you make some comment about bringing the limits on the fitting range in, say to 0.1 and 2.0, otherwise the fit isn't successful!
![Screenshot 2021-05-17 at 16 06 02](https://user-images.githubusercontent.com/55980573/118511666-0a0bff80-b72a-11eb-93dc-05c4e9200999.png)